users/services/registration.py

```diff
@@ -1,13 +1,12 @@
-import imp
-import uuid
 from .utils import validate_phone_number
 from ..models import OdiUser, Company, CompanyOneTimeLinkCode, Assessor, Assessee
 from one_day_intern.exceptions import InvalidRegistrationException, EmailNotFoundException
 from . import utils
+import uuid
 
 
 def validate_user_registration_data(request_data):
-    email = request_data.get('email')
+    email = request_data.get('email').lower()
```

Comment:
Hello, Jo. When we lower the email, it will become case insensitive now, cmiiw?

Reply:
Alex, you are correct. We will treat email in accordance with Gmail's convention rather than the RFC, which focuses solely on the domain's case insensitivity.

Reply:
Okay, thanks for the clarification Johanes.